### ISSUE TITLE
Implement Model Policy

### ISSUE THREAD

**Issue by dabooz** [open-horizon/anax#945]
- [x] Update ESS and CSS to latest code from edge-sync-service, to pick up the model policy changes
- [x] Update MMS CLI to support policy
- [x] Implement basic CLI for MMS objects
- [x] Update MMS object destinations when services are deployed
- [x] Support object affinity based authorization
- [x] Support deleted objects
- [x] Update e2edev tests (deferred to issue #1008 )

**Comment by jiportilla** [comment]
Waiting on Haifa code.
